hack/crds-sync.sh

```diff
@@ -17,3 +18,18 @@ done
 
 #this one goes in manifests rather than install, but should it? 
 cp "vendor/github.com/openshift/api/machineconfiguration/v1/0000_80_controllerconfig.crd.yaml" "manifests/controllerconfig.crd.yaml"
+cp "vendor/github.com/openshift/api/machineconfiguration/v1alpha1/0000_80_machineconfignode-TechPreviewNoUpgrade.crd.yaml" "manifests/0000_80_machine-config-operator_01_machineconfignode-TechPreviewNoUpgrade.crd.yaml" 
```

Comment:
wondering if there is a reason for not keeping it under install/ ?

Reply:
So, I am honestly not sure. I was noticing the CRD was not getting installed in a cluster where the FG is not enabled but it also WILL be installed in a cluster who has it enabled on installation. I think keeping it in both is the safe option. If the FeatureGate is not enabled, we still need to install the CRD to avoid API errors, this is why it is in manifests.

So basically, I think we need both